File: push/src/main/java/com/xiaomi/mipush/sdk/stat/upload/IDbPathGetter.java
package com.xiaomi.mipush.sdk.stat.upload;

import android.content.Context;
import java.util.List;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/stat/upload/IDbPathGetter.class */
public interface IDbPathGetter {
    String getPath(Context context, String str);

    List<String> getPathList(Context context);
}

File: legacy/src/main/java/com/xiaomi/mipush/sdk/stat/upload/IDbPathGetter.kt
package com.xiaomi.mipush.sdk.stat.upload
import io.github.magisk317.mipush.protocol.model.*

import android.content.Context

interface IDbPathGetter {
    fun getPath(context: Context, str: String): String

    fun getPathList(context: Context): List<String>
}
